controllers/admin/AdminProductsController.php

```diff
@@ -4106,10 +4108,21 @@ protected function addCarriers($product = null)
         }
     }
 
-    public function ajaxProcessaddProductImage()
+    /**
+     * Ajax process upload images
+     *
+     * @param int|null $idProduct
+     * @param string $inputFileName
+     * @param bool $die If method must die or return values
+     *
+     * @return array
+     */
+    public function ajaxProcessaddProductImage($idProduct = null, $inputFileName='file', $die = true)
```

Comment:
should be ``ajaxProcessAddProductImage`` according to the previous method name :)